packages/storybook-react/src/stories/Blockquote.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Blockquote, Paragraph } from '@utrecht/component-library-react/dist/css-module';
+import readme from '@utrecht/components/blockquote/README.md?raw';
+import tokensDefinition from '@utrecht/components/blockquote/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const meta = {
+  title: 'React Component/Blockquote',
+  id: 'react-blockquote',
+  component: Blockquote,
+  tags: ['autodocs'],
+  args: {
+    attribution: undefined,
+    children: (
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+        laborum
+      </Paragraph>
+    ),
+  },
+  argTypes: {
+    lead: {
+      name: 'lead',
```

Comment:
These argTypes are left from a copied Paragraph story I think? I would expect attribution to be mentioned here instead?